### DISCUSSION TITLE
feature: ability to zoom in and out in the editor

### DISCUSSION THREAD

**Discussion by da-rulez** [AscensionGameDev/Intersect-Engine#1554] [Ideas]
### Description

A button in the editor to zoom out, and in, when editing maps. 

### Intended Use-Case

You'd be able to use this button to make more tiles visible. Especially useful when working with bigger tilesizes where you only have like a quarter of the map visible at any time. 

### Duplicate Check

- [X] This enhancement idea is not a duplicate to the best of my knowledge.

**Comment by Zano742** [comment]
Why only limit it to the editor? It would be nice to be able to zoom in when inside a smaller house to 80% of the screen isn't just black void space. Even better, set a map default zoom level so when players enter a certain map like a house, the game client automatically zooms them in! :)

**Reply by pandinocoder** [reply]
Not all players will appreciate the automatic zooming (even if it had no issues), but we cannot zoom to non-powers of 2 or else seams appear in the rendering or we get smeared scaling artifacts (or both! this looks _horrendous_). The auto-zoom as a result would not be usable in most cases, though maybe a camera update in the future can give developers the ability to zoom in during events.
